pkg/proc/gdbserial/undo.go

```diff
@@ -327,6 +355,7 @@ func undoWhen(conn *gdbConn) (string, error) {
 		progress = ((bbcount - bbcount_min) * 100) / (bbcount_max - bbcount_min)
 	}
 
-	result := fmt.Sprintf("[replaying %d%% %d:0x%x]", progress, bbcount, pc)
+	progress_fmt := fmt.Sprintf("%d%%", progress)
+	result := fmt.Sprintf("[replaying %s %s]", progress_fmt, undoTimeString(bbcount, pc))
```

Comment:
I wonder if we could find a better name than "progress" here, since elsewhere we use the term for progress of a time-travel or reverse-execution operation. Perhaps `history_percentage` or something along those lines?

Reply:
Yes, `history_percentage` (well, might shorten to `history_perc`) isbetter.

Reply:
`history_perc` it is, for now at least.